build/set-env.ps1

```diff
@@ -64,4 +64,7 @@ Get-ChildItem @(
     "Env:\CRATE_OUTDIR",
     "Env:\WHEEL_OUTDIR",
     "Env:\DOCS_OUTDIR"
- ) | Format-Table
\ No newline at end of file
+ ) | Format-Table
+
+Write-Host "PATH:"
```

Comment:
Is this for diagnostics during builds?

Reply:
Yes, for diagnostic and debugging purposes, to analyze the build failure reasons.